app/controllers/Application.scala

```diff
@@ -23,6 +23,14 @@ class Application(
   import actionRefiners._
 
   implicit val ar = assets
+
+  private def applyCircles(queryParam: String, id: String, js: String, modifiedId: String, modifiedJs: String): (String, String) = {
+    queryParam match {
+      case "circles-garnett" => (modifiedId, modifiedJs)
+      case _ => (id, js)
```

Comment:
Personally, I don't really trust myself to use the values in the correct order when I extract them from a tuple where both values are of the same type. As a personal style, I'd usually introduce a class in this case. Nonetheless, I acknowledge that you might find this overkill.